drafts/liberty-dev-mode-gradle.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Rapid application development with Gradle and Open Liberty's dev mode"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/28316667
+author_github: https://github.com/rzgry
+seo-title: Rapid application development with Gradle and Open Liberty's dev mode - OpenLiberty.io
+seo-description: Develop Java applications quicker using Open Liberty development mode with Gradle.
+blog_description: "Develop Java applications quicker using Open Liberty development mode with Gradle."
+---
+= Open Liberty development mode for Gradle
+Ryan Zegray <https://github.com/rzgry>
+
+Open Liberty development mode, or dev mode, now works with Gradle projects. Open Liberty dev mode allows you to develop applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support. Your code is automatically compiled and deployed to your running server, making it easy to iterate on your changes. You can run tests on demand or even automatically so that you can get immediate feedback on your changes. You can also attach a debugger at any time to debug your running application.
```

Comment:
```suggestion
Open Liberty development mode, or dev mode, now works with Gradle projects. Open Liberty dev mode allows you to develop cloud-native Java applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support. Your code is automatically compiled and deployed to your running server, making it easy to iterate on your changes. You can run tests on demand or even automatically so that you can get immediate feedback on your changes. You can also attach a debugger at any time to debug your running application.
```